src/status_im/ui/screens/wallet/request/views.cljs

```diff
@@ -0,0 +1,65 @@
+(ns status-im.ui.screens.wallet.request.views
+  (:require-macros [status-im.utils.views :as views])
+  (:require
+    [re-frame.core :as re-frame]
+    [status-im.components.react :as react]
+    [status-im.components.qr-code :refer [qr-code]]
```

Comment:
No `:as` here?

Reply:
yeah, `qr-code/qr-code` maybe we can use refer for such cases